### ISSUE TITLE
About Loss Value

### ISSUE THREAD

**Issue by jnanliu** [kmkurn/pytorch-crf#124]
Thanks for the great package.  In my work, I use the -log_likehood score as the loss, however, when training after several batches, the value of -log_likehood becomes negative. Is this normal?
